homeassistant/components/cloud/__init__.py

```diff
@@ -288,9 +288,11 @@ async def async_startup_repairs(_: datetime) -> None:
     loaded = False
     stt_platform_loaded = asyncio.Event()
     tts_platform_loaded = asyncio.Event()
+    stt_tts_entities_added = asyncio.Event()
     hass.data[DATA_PLATFORMS_SETUP] = {
```

Comment:
This should really be a dataclass instance instead. OK to do in a follow-up if it adds too much noise to this PR.

Reply:
Yeah, I'll clean it up in the next PR.